client/gateway/dappchain_gateway.go

```diff
@@ -93,6 +108,39 @@ func (tg *DAppChainGateway) AddContractMapping(from common.Address, to loom.Addr
 	return err
 }
 
+// AddTronContractMapping same as AddContractMapping but for Tron
+func (tg *DAppChainGateway) AddTronContractMapping(from common.Address, to loom.Address,
+	creator *client.Identity, contractTxHash string) error {
```

Comment:
This doesn't need the `contractTxHash` anymore does it?

Reply:
not anymore